### ISSUE TITLE
D:

### ISSUE THREAD

**Issue by saltacc** [7eu7d7/anime-ai-detect-fucker#2]
I never expected this to get big enough for somebody to write a bypass lol. (also, you forgot transformers in the requirements.txt)

**Comment by IrisRainbowNeko** [comment]
At current stage adversarial attacks are effective against high level vision models and there is no well theoretical solution to defend those attacks. So it is not possible to use AI to detect AI-generated images. Why not use those time and computing resources to do something more meaningful? Don't waste it on such problems.

**Comment by saltacc** [comment]
@7eu7d7 Yeah I knew it could be bypassed easily without anything I can do about it when I created it. The goal of this was an AI image detector to remove AI images from training data sets. I made this in around an hour and it trained while I slept, never really gave it too much thought lol. I am using my compute for something more meaningful though, I'm one of the developers of Waifu Diffusion. I made this mainly just in case we needed to remove AI generated images currently or in the future from our dataset.
